### DISCUSSION TITLE
Link Convertion  Dash to hls

### DISCUSSION THREAD

**Discussion by Sphinxroot** [iptv-org/community#314] [Show and tell]
https://robinhood.stream/ plenty of link to grab

Those wont play for some reason 

https://pe-ak-lp02a-9c9media.akamaized.net/live/DiscoveryChan/p/dash/20000001/07836f4eb9ce89b2/manifest.mpd
https://pe-ak-lp01a-9c9media.akamaized.net/live/CKY/p/dash/20000001/cde0424638a615e6/manifest.mpd
https://pe-ak-lp01a-9c9media.akamaized.net/live/CIVI/p/dash/20000001/832d3547fbcdaed1/manifest.mpd
https://pe-ak-lp02a-9c9media.akamaized.net/live/ctvnews/p/dash/20000001/2f09044e4a899f70/manifest.mpd
https://pe-ak-lp01a-9c9media.akamaized.net/live/CFCF/p/dash/20000001/71e7c5f500f495f3/manifest.mpd
https://pe-ak-lp02a-9c9media.akamaized.net/live/CKVR/p/dash/20000001/a889a00582c22aa9/manifest.mpd
https://pe-ak-lp01a-9c9media.akamaized.net/live/CFTO/p/dash/20000001/9117ec94ec99c195/manifest.mpd
https://pe-ak-lp02a-9c9media.akamaized.net/live/CTVSciFi/p/dash/20000001/4831ddda5a60154a/manifest.mpd
https://pe-ak-lp02a-9c9media.akamaized.net/live/Gusto/p/dash/00000001/5faeeb15356186d7/manifest.mpd
https://pe-ak-lp02a-9c9media.akamaized.net/live/Bravo/p/dash/20000001/b8b9d389e89e212e/manifest.mpd
https://pe-ak-lp02a-9c9media.akamaized.net/live/CTVComedy/p/dash/20000001/8dd7e255d7ada118/manifest.mpd
https://pe-fa-lp01a.9c9media.com/live/TMNWest1/p/dash/20000001/f951e2226ce07bf8/manifest.mpd
https://pe-fa-lp04a.9c9media.com/live/TMNEast3/p/dash/20000001/c01fd79a254fe629/manifest.mpd
https://pe-fa-lp04a.9c9media.com/live/TMNEast2/p/dash/20000001/18d3a96d9c503de4/manifest.mpd
https://pe-fa-lp03a.9c9media.com/live/TMNEast1/p/dash/20000001/c784f76b65daf97f/manifest.mpd
https://pe-ak-lp01a-9c9media.akamaized.net/live/TSN5/p/dash/20000001/087b8043dce361c0/manifest.mpd
https://pe-ak-lp04a-9c9media.akamaized.net/live/TSN4/p/dash/20000001/8caedc40dcc49339/manifest.mpd
https://pe-ak-lp02a-9c9media.akamaized.net/live/TSN3/p/dash/20000001/8668457937758888/manifest.mpd
https://pe-ak-lp02a-9c9media.akamaized.net/live/TSN2/p/dash/20000001/c3e23fb53bc40cb2/manifest.mpd
https://pe-ak-lp04a-9c9media.akamaized.net/live/TSN/p/dash/20000001/9d50f71ec5db29e6/manifest.mpd
https://pe-ak-lp02a-9c9media.akamaized.net/live/DiscVelocity/p/dash/20000001/6fa8c8af6a0e721d/manifest.mpd
https://pe-ak-lp02a-9c9media.akamaized.net/live/InvestDisc/p/dash/20000001/dc06a49abc9b86d2/manifest.mpd

this one play ok
https://pe-ak-lp04a-9c9media.akamaized.net/live/NOOVO/p/dash/00000001/f481c583dbd06b6c/manifest.mpd

**Comment by BellezaEmporium** [comment]
I got a login page, won't be easy.

**Reply by Sphinxroot** [reply]
demo:acct

**Reply by BellezaEmporium** [reply]
Yup, works. Interesting thing...
